src/main/java/com/ldtteam/blockout/views/DropDownList.java

```diff
@@ -167,7 +163,7 @@ private void onButtonClickedFromList(@NotNull final Button buttonIn)
         final Text idLabel = buttonIn.getParent().findPaneOfTypeByID("id", Text.class);
         if (idLabel != null)
         {
-            final int index = Integer.parseInt(idLabel.getTextAsString());
+            final int index = Integer.parseInt(idLabel.text.get().getString());
```

Comment:
this is kinda not intuitive to get the first row of text

Reply:
I've seen it used before for fetching first elements, but it is more intuitive when you consider that the same function is used for text with only one line.

Reply:
wouldn't a text.get(0) be more intuitive?

Reply:
The code allows for that, but again this is clarity for multi-line vs single-line. This way implies that I only ever expect to work with the first line. But yes can add the index if you prefer.